packages/webpack5/package.json

```
@@ -17,27 +17,27 @@
 		"prepack": "npm run test && npm run build && npm run copy-stubs && chmod +x dist/bin/index.js"
 	},
 	"dependencies": {
-		"@babel/core": "^7.0.0",
-		"@pmmmwh/react-refresh-webpack-plugin": "^0.5.2",
+		"@babel/core": "^7.19.3",
```

```suggestion
		"@babel/core": "^7.0.0",
```

Keeping the wider range should still resolve to 7.19.3 but allow deduping dependencies where a different babel/core is installed instead of producing broken builds or duplicate dependencies...